schemas/groups/propulsion.json

```diff
@@ -213,6 +213,26 @@
               "description": "Average fuel rate of consumption",
               "$ref": "../definitions.json#/definitions/numberValue",
               "units": "m3/s"
+            },
+            "vacuum": {
+              "description": "vacuum measured at the fuel filter - an indication of the state of the fuel filter. The higher the vacuum, the more clogged the filter.",
+              "$ref": "../definitions.json#/definitions/numberValue",
+              "units": "Pa"
+            },
+            "currentUsed": {
+              "description": "Amperes used since last reset. Resetting is at user discretion",
```

Comment:
Perhaps the correct term would be "Amp hours used since last reset"? I think this came from someone who has an electric motor for propulsion, and I tried to translate his request for "current" into the schema. But he probably meant "current being consumed right now". I'll change it.